osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerMatchFooter.cs

```diff
@@ -19,7 +20,13 @@ public class MultiplayerMatchFooter : CompositeDrawable
 
         public readonly Bindable<PlaylistItem> SelectedItem = new Bindable<PlaylistItem>();
 
+        public Action OnToggleReady
```

Comment:
wow, this is weird syntax. what happens if you call the `get`? usually we fill it in with an exception.

Reply:
You can't lest you get a compile-time error. I already kinda sorta used it once (see `IBarLine`). Can expose the getter, but not sure there's too much point?

Reply:
Seems fine as long as it's compile time error.